src/v/features/feature_table.h

```diff
@@ -109,6 +95,23 @@ inline const std::unordered_set<std::string_view> retired_features = {
   "idempotency_v2",
   "transaction_partitioning",
   "lightweight_heartbeats",
+  "serde_raft_0",
+  "license",
+  "raft_improved_configuration",
+  "transaction_ga",
+  "raftless_node_status",
+  "rpc_v2_by_default",
+  "cloud_retention",
+  "node_id_assignment",
+  "replication_factor_change",
+  "ephemeral_secrets",
+  "seeds_driven_bootstrap_capable",
+  "tm_stm_cache",
+  "kafka_gssapi",
+  "partition_move_revert_cancel",
+  "node_isolation",
+  "group_offset_retention",
```

Comment:
I don't think you want to include `node_isolation`, `group_offset_retention` here.